### ISSUE TITLE
No response after awaiting client.requestOneTimeLoginKey in dev mode after reload (IOS only)

### ISSUE THREAD

**Issue by ruscoder** [voximplant/react-native-voximplant#45]
I got stuck with a method `client.requestOneTimeLoginKey` while I'm developing the application.

The example of code:
```
const client = Voximplant.getInstance();
try {
    await client.disconnect();
} catch (err) {}
await client.connect();
await client.requestOneTimeLoginKey(fullUsername);
```

Everything is OK when I first run this code on application startup, 
but after reload (cmd+R) `client.requestOneTimeLoginKey` has no response.

Also, when you execute this code two times after reload, you'll receive a response to both requests.

**Comment by YuliaGrigorieva** [comment]
Hello!

Thank you for your report!

Could you please provide the following information:
1. RN version 
2. react-native-voximplant sdk version

Best regards,
Yulia Grigorieva

**Comment by ruscoder** [comment]
Hello, @YuliaGrigorieva! 
1. RN: 0.56.0
2. react-native-voximplant 1.1.1

**Comment by YuliaGrigorieva** [comment]
Hello!

Thank you for the information provided. I was able to reproduce the issue. We will include the fix to the nearest Voximplant React Native SDK release.

Best regards,
Yulia Grigorieva

**Comment by YuliaGrigorieva** [comment]
Hello!

We have just released Voximplant React Native SDK 1.2.0 that includes the fix for this issue.
Please update SDK in your project.

Best regards,
Yulia Grigorieva
